Tasks/DownloadPackageV1/Tests/L0DownloadNpmPackageDownloadError.ts

```diff
@@ -0,0 +1,56 @@
+import tmrm = require("azure-pipelines-task-lib/mock-run");
+import path = require("path");
+import { WebApiMock } from "./helpers/webapimock";
+
+let taskPath = path.join(__dirname, "..", "main.js");
+let outputPath: string = path.join(__dirname, "out", "packageOutput");
+let tempPath: string = path.join(__dirname, "temp");
+let zipLocation: string = path.join(tempPath, "singlePackageName.tgz");
+let tr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
+
+// Set inputs
+tr.setInput("packageType", "npm");
+tr.setInput("feed", "feedId");
+tr.setInput("view", "viewId");
+tr.setInput("definition", "6f598cbe-a5e2-4f75-aa78-e0fd08301a15");
+tr.setInput("version", "versionId");
+tr.setInput("downloadPath", outputPath);
+tr.setInput("extract", "true");
+tr.setInput("verbosity", "verbose");
+
+// Set variables.
+process.env["AGENT_TEMPDIRECTORY"] = tempPath;
+process.env["SYSTEM_TEAMFOUNDATIONCOLLECTIONURI"] = "https://abc.visualstudio.com/";
+process.env["AGENT_VERSION"] = "2.116.0";
+process.env["HOME"] = "/users/test";
+process.env["ENDPOINT_AUTH_SYSTEMVSSCONNECTION"] =
+    '{"scheme":"OAuth","parameters":{"AccessToken":"YWFtYWxsYWQ6ZXd0emE1bmN3MzN6c3lyM2NoN2prazUzejczamN6MnluNGtiNzd0ZXc0NnlhZzV2d3ZlcQ=="}}';
```

Comment:
Where did you get this access token? We shouldn't check in any access tokens

Reply:
@aasim do you have context as I think you're the most familiar with the DownloadPackage task? As @ByAgenT  mentioned above it looks like this is just the way we've set up the l0s.

Reply:
I just cleared variables that are not affecting test results for all tests in DownloadPackageV1, by the way.